developers.diem.com/README.md

```diff
@@ -62,7 +56,3 @@ Get the latest updates to our project by signing up to our [newsletter](https://
 ## License
 
 Diem Core is licensed as [Apache 2.0](https://github.com/diem/diem/blob/master/LICENSE)
-
-## Docs Directory
-
-The [docs directory](./docs/) contains the source files for Diem's Docusaurus documentation. See the [website directory](./website/) [README](./website/README.md) for additional information.
```

Comment:
I was just asking why these 3 lines were removed. 66-68.

Reply:
I removed it because we removed the other readme file. But perhaps I should keep the first sentence. Thoughts?

Reply:
Yeah, you can do that.